app/views/vendors/index.html.erb

```diff
@@ -1,76 +1,38 @@
 <div class="row">
-  <% @vendors.each do |vendor| %>
-
   <div class="large-4 small-12 columns">
 
     <img src="http://placehold.it/500x500&text=Logo">
 
     <div class="hide-for-small panel">
```

Comment:
Do you really want to hide these headers for small screens? That's what your `hide-for-small` class will do... it seems like it would be useful to have a header for this page even on small screens!